controllers/external-genieacs/cpe-models/flashify-model.js

```diff
@@ -1115,42 +1115,52 @@ const propertyBuilder = {
         || rfa('wifi5BandEdit', 'band20_mapping');
     }
 
-    // fallbackValue24/50 will be the first valid value on table24/50, just to
-    // not crash the app. This will soon be unnecessary because we will soon
-    // insert a "validWifiBands" permission
-    const fallbackValue24 = Object.values(table24).filter((v)=> v)[0];
-    const fallbackValue50 = Object.values(table50).filter((v)=> v)[0];
-
-    let hasValidValues24 = Object.values(table24)
+    // valid values can't be undefined or null or empty string
+    const validValues24 = Object.values(table24)
       .filter(
-        (v) => v !== undefined && v.toString() !== 'null' && v.length > 0,
-      ).length > 0;
+        (v) => (v !== undefined && v !== null) ||
+              (typeof(v) === 'string' && v.length > 0),
+    );
```

Comment:
A validação não vai funcionar, ele só vai verificar se é `string` se for `undefined` ou `null`. Acredito que queira algo do tipo:


```suggestion
    const validValues24 = Object.values(table24).filter(
        (v) => (v !== undefined && v !== null && (
            (typeof v === 'string' && v.length > 0) ||
            (typeof v === 'number')
        )),
    );
```

Ou melhor ainda:


```suggestion
    const validValues24 = Object.values(table24).filter(
        (v) => (
			(typeof v === 'string' && v.length > 0) ||
			typeof v === 'number'
		),
    );
```

Reply:
Usei a segunda opção.